packages/engine-backend/router/pipelineRouter.ts

```diff
@@ -36,6 +36,21 @@ export const pipelineRouter = trpc.router({
     .mutation(async ({ctx, input: {id, ...input}}) =>
       ctx.services.patchReturning('pipeline', id, input),
     ),
+  createPipeline: protectedProcedure
+    .meta({openapi: {method: 'POST', path: '/core/pipeline', tags}})
+    .input(
+      zRaw.pipeline.pick({
+        id: true,
+        metadata: true,
+        disabled: true,
+        sourceId: true,
+        destinationId: true,
+      }),
+    )
+    .output(zRaw.pipeline)
+    .mutation(async ({ctx, input: {id, ...input}}) =>
+      ctx.services.patchReturning('pipeline', id, input),
```

Comment:
@jatinsandilya does this work? `id` might be missing for create calls.

Reply:
@tonyxiao it does. Was required for Twenty integration. The `id` is passed from the request here.